### PR TITLE
Content issues

### LINKED ISSUE #225: Mismatched tooltips
I'm looking at the tooltips on the right hand side of the first equation under "[State vectors](https://learn.qiskit.org/course/introduction/describing-quantum-computers#describing-1-0)" on the "Describing quantum computers" page.  It seems that the tooltips for the first two elements of this vector are somehow being shadowed by the ones from [entangled-states.ipynb](https://github.com/qiskit-community/platypus/blob/373420808119361db9871cc6966b55b05dec6713/notebooks/intro/entangled-states.ipynb?short_path=57ecba6#L143-L148), even though [they are defined correctly in the actual notebook that generates that page](https://github.com/qiskit-community/platypus/blob/373420808119361db9871cc6966b55b05dec6713/notebooks/intro/describing-quantum-computers.ipynb?short_path=6d7f9e6#L79-L84).  Presumably, this is happening because the identifiers `_amp-0` and `_amp-1` are being used in both places.

### LINKED ISSUE #233: [1-1.1] Introduction-New Qiskit Textbook beta
### Describe the issue

In the Introduction of the new beta version of the Qiskit textbook the sentence after the quick exercise says that "But if you chose the second one, it shows...". This is not always correct because the order of options in the exercise gets shuffled.  
Qiskit-Introduction-Bug

### Describe what you expect to see

We can have it as "But if you chose it as the smallest unit of information, it shows...". 

Or  

"But if you chose the first/second/third*, it shows...".

*Choose one among  first/second/third, depending on the correct option in the quick exercise.

### PR DESCRIPTION
**Pull request by frankharkins**
Fixed some minor problems.
- Fixes https://github.com/Qiskit/qiskit.org/issues/2352
- Fixes #233 
- Fixes #225

### CODE REVIEW

File: notebooks/ch-states/introduction.ipynb [hunk 1]
```diff
@@ -38,39 +38,13 @@
...
-    "<!-- ::: .option -->\n",
-    "\n",
-    "Something you put in a horse's mouth.\n",
-    "\n",
-    "<!-- ::: -->\n",
-    "\n",
-    "<!-- ::: -->\n",
-    "\n",
-    "<!-- ::: -->\n"
+    "- The smallest unit of information: either a ```0``` or a ```1```.\n",
```
Comment (techtolentino) [inline notebooks/ch-states/introduction.ipynb:45]: @frankharkins - I'm curious if the intention here is to remove the interactive quiz. I'm seeing this in the preview branch:

Screen Shot 2021-12-20 at 10 48 10 AM

Are you seeing the same?

https://platypus-pr-237.ecy0akwlcpw.us-south.codeengine.appdomain.cloud/course/ch-states/introduction

Comment (frankharkins) [reply notebooks/ch-states/introduction.ipynb:45]: Yeah that's right; technically all are correct, but our quiz only supports one correct answer so I decided to remove it here. This page is really superseded by the intro course and we might want to think about removing it from platypus at some point in the future.

**Review by techtolentino — COMMENTED** [review]
Thanks for the explanation on the quiz, @frankharkins 

the changes look good to me... the branch just needs to be updated.

**Review by vabarbosa — APPROVED** [review]
LGTM

**Pull request merged**